src/app/api/lista-nominal/[list]/[municipio_id_sus]/route.ts

```
@@ -0,0 +1,117 @@
+
+      sortingOptions.forEach((option) => {
+          const [field, order] = option.split(':');
+          validateSortOrder(order);
+          // será substituido por consulta no banco de dados
+          responseData = [...sortData({
+            data: responseData,
+            field,
+            sortOrder: order as SortOrder,
+          })];
+        });
+    }
```

Biome (1.9.4)

[error] Prefer for...of instead of forEach.

forEach may lead to performance issues when working with large arrays. When combined with functions like filter or map, this causes multiple iterations over the same type.

(lint/complexity/noForEach)